pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/io/buffer/Bytes.java

```diff
@@ -334,13 +334,10 @@ public int read() throws IOException {
     public String toString() {
         // build string
         StringBuilder sb = new StringBuilder();
-        sb.append("Bytes[");
         for (long i = 0; i < length(); i++) {
             int v = getByte(i) & 0xFF;
-            sb.append(v);
-            if (i < (length()-1)) sb.append(',');
+            sb.append(Integer.toHexString(v).toUpperCase());
         }
-        sb.append(']');
         return sb.toString();
```

Comment:
I am not sure on the raw hex format, here are some other options.
1. `Bytes[255, 255, 255, 255]`  - What we have today that I thing is too long winded
2. `FFFF` - just plain HEX
3. `0xFFFF`  - not sure where this style is from
4. `[FFFF]` in square brackets
5. `Bytes[FFFF]` with class name

Think I like (3) best, what do you think?